Justfile

```diff
@@ -1,16 +1,43 @@
-set positional-arguments
+set positional-arguments := true
 
 _help:
-  @just -l
-
-schemas:
-  #!/bin/bash
-  set -euo pipefail
-  
-  source_dir="hosted/json-schemas" 
-  dest_dir=".schemas"
-
-  mkdir -p $dest_dir
-  rm -rf $dest_dir/*
-  cp -R $source_dir/* $dest_dir/
-  echo "Schema files successfully copied to $dest_dir/"
\ No newline at end of file
+    @just -l
+
+clean:
+    #!/usr/bin/env bash
+    find hosted -type f -name "*.tmp" -delete
+
+format: lint
+    #!/usr/bin/env bash
+    find hosted -type f -name "*.json" | while read file; do
+      jq --indent 2 . $file > $file.json.tmp
+      mv $file.json.tmp $file
+      printf "formatted: %s\n" $file
+    done
+
+lint: clean
```

Comment:
should we add a github workflow which runs `just lint` such that PRs can't be merged if linting fails?

Reply:
ohh good idea!

i didn't spend too much time digging into this but i thought we generated the schemas somewhere so they'd always be valid? if these are manually crafted then that makes a ton of sense and i can definitely add that

Reply:
added a lint (on pull_request) and format (on push to `main`) [5e903a5](https://github.com/TBD54566975/tbdex/pull/364/commits/5e903a59832557bfb0a609e99263f5ef50408e80)

Reply:
looks good!